3.12/graphs-general-graphs.md

```diff
@@ -7,16 +7,60 @@ General Graphs
 ==============
 
 This chapter describes the [general-graph](graphs.html) module.
-It allows you to define a graph that is spread across several edge and document collections.
-This allows you to structure your models in line with your domain and group them logically in collections giving you the power to query them in the same graph queries.
-There is no need to include the referenced collections within the query, this module will handle it for you.
+It allows you to define a graph that is spread across several edge and document
+collections.
+This allows you to structure your models in line with your domain and group
+them logically in collections giving you the power to query them in the same
+graph queries.
+There is no need to include the referenced collections within the query, this
+module will handle it for you.
 
 New to ArangoDB? Take the free
 [ArangoDB Graph Course](https://www.arangodb.com/arangodb-graph-course){:target="_blank"}
 for freshers.
 
-Three Steps to create a graph
------------------------------
+## Getting started
+
+### Create a GeneralGraph using the Web Interface
+
+The Web Interface (also called Web UI) allows you to easily create and manage
+GeneralGraphs. To get started, follow the steps outlined below.
+
+1. In the main page of the Web Interface, go to the left sidebar 
+   menu and select the **Graphs** tab.
+2. To add a new graph, click **Add Graph**.
+3. In the **Create Graph** dialog that appears, select the
+   **GeneralGraph** tab.
+4. Fill in the following fields:
+   - For **Name**, enter a name for the GeneralGraph.
+   - For **Shards**, enter the number of shards the graph is using.
+   - For **Replication factor**, enter the total number of
+     desired copies of the data in the cluster.
+   - For **Write concern**, enter the total number of copies
+     of the data in the cluster required for each write operation.
+5. Define the relation(s) on the GeneralGraph:     
+   - For **Edge definition**, insert a single non-existent name to define
+     the relation of the graph. This automatically creates a new edge
+     collection, which is displayed in the **Collections** tab of the
+     left sidebar menu.
+     {% hint 'tip' %}
+     To define multiple relations, press the **Add relation** button.
+     To remove a relation, press the **Remove relation** button.
+     {% endhint %}
+   - For **fromCollections**, insert a list of vertex collections
+     that contain the start vertices of the relation.
+   - For **toCollections**, insert a list of vertex collections that
+     contain the end vertices of the relation.
+6. If you want to use vertex collections that are part of the graph
+   but not used in any edge definition, you can insert them via
+   **Orphan collections**.
+7. Click **Create**. 
+8. Open the graph and use the functions of the Graph Viewer to visually
```

Comment:
Click the card of the newly created graph?

Reply:
Yep, changed it to "card".